pyproject.toml

```diff
@@ -41,13 +41,17 @@ issues = "https://github.com/instructlab/instructlab/issues"
 
 [tool.setuptools_scm]
 version_file = "src/instructlab/_version.py"
+# do not include +gREV local version, required for Test PyPI upload
+local_scheme = "no-local-version"
 
 [tool.setuptools]
 package-dir = {"" = "src"}
 
 [tool.setuptools.dynamic]
 dependencies = {file = ["requirements.txt"]}
-optional-dependencies.hpu = { file = ["requirements-hpu.txt"]}
+# Intel Gaudi needs a package from git, but PyPI does not support packages
+# with direct dependencies.
+# optional-dependencies.hpu = { file = ["requirements-hpu.txt"]}
```

Comment:
is this a temporary measure? Why should we keep it as a comment?

Reply:
It's a temporary measure. Habana has is maintaining several forks. There are plans to merge the forks upstream over the course of the year.